src/commands/diagcmds.cpp

```diff
@@ -15,7 +15,8 @@ namespace
 	}
 	std::shared_ptr<value> diag_tickTime_(virtualmachine* vm)
 	{
-		long r = std::chrono::duration_cast<std::chrono::milliseconds>(virtualmachine::system_time().time_since_epoch()).count();
+		// Easily fits in a long
```

Comment:
Actually I realized this is wrong for x86 build, it WON'T easily fit in a long! However I guess overflow is pretty much expected behavior here anyway, it isn't actually for working out the time...